tests/cases/fourslash/refactorConvertToEs6Module_export_moduleDotExports.ts

```diff
@@ -0,0 +1,29 @@
+/// <reference path='fourslash.ts' />
+
+// @allowJs: true
+
+// @Filename: /a.js
+/////*a*/module/*b*/.exports = function() {}
+////module.exports = function f() {}
+////module.exports = class {}
+////module.exports = class C {}
+////module.exports = 0;
+////
+////module.exports = require("./b");
+
+goTo.select("a", "b");
+edit.applyRefactor({
+    refactorName: "Convert to ES6 module",
+    actionName: "Convert to ES6 module",
+    actionDescription: "Convert to ES6 module",
+    newContent: `export default function() { }
+export default function f() { }
+export default class {
+}
+export default class C {
+}
+export default 0;
+
+export * from "./b";
+export { default } from "./b";`,
```

Comment:
this seems wrong to me.. or at least having both exports seem wrong.. i think you either want one or the other.. and to a big extend it depends on what the shape of `"./b"` is.  if `"./b"` does not have a default export, this statement is an error under ES6 semantics.

I would say we should do one or the other, i.e. always do `export * from "./b"` unless `",/b"` has only one export that happens to be `default`.